src/components/Notification/Notification.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import { Transition } from 'react-transition-group'
+
+import { Button, Icon } from '../../../src'
+
+const propTypes = {
+  item: {
+    show: PropTypes.bool,
+    key: PropTypes.string,
+    type: PropTypes.string,
+    position: PropTypes.string,
+    isDismissable: PropTypes.bool,
+    isWithIcon: PropTypes.bool,
+    icon: PropTypes.string,
+    title: PropTypes.string,
+    text: PropTypes.string
+  },
+  close: PropTypes.func
+}
+
+class Notification extends React.Component {
+  render() {
+    if (!this.props || !this.props.item) return ''
+    const {
+      show,
+      key,
+      type = 'success',
```

Comment:
Rather than default values in destruturing, it is better to use the `defaultProp` value as in other documents. This let the documentation task pick that up and show it to the fianl user.